backend/src/main/scala/akka/viz/events/LightSnapshot.scala

```diff
@@ -11,8 +11,9 @@ case class LightSnapshot(
     receivedFrom: Set[(String, String)] = Set()
 ) {
 
-  implicit def ref2String(r: ActorRef): String = r.path.toSerializationFormat
-  implicit def refPair2StringPair(pair: (ActorRef, ActorRef)): (String, String) = (ref2String(pair._1), ref2String(pair._2))
+  import Helpers.actorRefToString
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/lustefaniak/akka-viz/file/2004142088/issues/source?bid=3121282&fileBranchId=3122179#l14)